src/lib/gocardless/client.ts

```
@@ -0,0 +1,28 @@
+import NordigenClient from 'nordigen-node';
+import { cache } from 'react';
+
+import type { GocardlessError } from './types';
+
+declare global {
+  // eslint-disable-next-line no-var
+  var gocardlessClient: NordigenClient | undefined;
+}
+
+const gocardlessClient =
```

Biome (1.9.4)

[error] Shouldn't redeclare 'gocardlessClient'. Consider to delete it or rename it.

'gocardlessClient' is defined here:


(lint/suspicious/noRedeclare)